.go-version

```diff
@@ -1 +1 @@
-1.21.6
+1.22.6
```

Comment:
We should probably bump this to the latest, otherwise when you try to release the plugin binary security scanner will complain.